spyder_kernels/customize/code_runner.py

```diff
@@ -0,0 +1,593 @@
+#
+# Copyright (c) 2009- Spyder Kernels Contributors
+#
+# Licensed under the terms of the MIT License
+# (see spyder_kernels/__init__.py for details)
+# -----------------------------------------------------------------------------
+#
+# Spyder magics
+#
+
+import ast
+import bdb
+import builtins
+import cProfile
```

Comment:
```suggestion
from contextlib import contextmanager
import cProfile
from functools import partial
```

Organize imports in alphabetical order.